ethereumj-core/src/main/java/org/ethereum/db/PeerSource.java

```diff
@@ -18,10 +18,7 @@
 package org.ethereum.db;
 
 import org.apache.commons.lang3.tuple.Pair;
-import org.ethereum.datasource.DataSourceArray;
-import org.ethereum.datasource.ObjectDataSource;
-import org.ethereum.datasource.Serializer;
-import org.ethereum.datasource.Source;
+import org.ethereum.datasource.*;
```

Comment:
imports: LevelDbDataSource unused
do we use * imports?

Reply:
no, we don't